docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -67,6 +67,10 @@ There are two primary topologies within many event-driven architectures:
 
   Another challenge with asynchronous communication is data loss. If any of the components crashes before successfully processing and handing over the event to its next component, then the event is dropped and never makes it into the final destination. To minimize the chance of data loss, persist in-transit events and remove or dequeue the events only when the next component has acknowledged the receipt of the event. These features are usually known as _client acknowledge mode_ and _last participant support_.
 
+- Simplicity and testability
+
+ Event-driven architectures can have unpredictable and dynamic event flows, leading to many possible scenarios. This complexity can create significant challenges when it comes to testing.
```

Comment:
Okay, how do you address this?  All of the other items here have a quick pointer in a general direction on how someone might address this situation.  We can't just burden the reader with this information and not give any hints on how to mitigate.

Reply:
Fair point. AFAIK, that's the million dollar question. I know some intelligent people are working on this but this space is evolving. I'm not sure whether I have any good answer on how to mitigate them. But given the authority of Azure doc, I suggest to keep this point here to keep readers informed what they are subscribing for; because EDA is not a one-size fit all solution. And users shouldn't opt for a style just because it's cool. However, I leave the decision (keep or discard this content), up to you.

Reply:
Given that this space is evolving, I would suggest to keep it and we can later add some solution (whenever something good comes up). This way we can keep the doc "evolutionary" :) but i leave it up to you.

Reply:
We'd like to do better than places that put extra burden on our learners. Sometimes it's okay to burden our learners, but in this case I'd like to burden them and point them in a meaningful direction right away.  Please add some supporting content.

Reply:
Fair point. I'm afraid I cant add a high quality content on this topic any further, at least for now. We can close this for now.